classes/helper/HelperList.php

```diff
@@ -515,7 +515,7 @@ public function displayDefaultLink($token = null, $id, $name = null)
 			self::$cache_lang['Default'] = $this->l('Default', 'Helper');
 
 		$tpl->assign(array_merge($this->tpl_delete_link_vars, array(
```

Comment:
Hi,

Maybe it will be good to create a different variable, like tpl_default_link_vars, do you think so ?

Best regards.